app/src/cc/arduino/contributions/packages/ui/ContributedPlatformTableCellJPanel.java

```diff
@@ -149,6 +165,25 @@ public ContributedPlatformTableCellJPanel() {
     add(Box.createVerticalStrut(15));
   }
 
+  // same function as in ContributedLibraryTableCellJPanel - is there a utils file this can move to?
+  private String setButtonOrLink(JButton button, String desc, String label, String url) {
+    boolean accessibleIDE = PreferencesData.getBoolean("ide.accessible");
+    String retString = desc;
+
+    if (accessibleIDE) {
+      button.setVisible(true);
+      button.addActionListener(e -> {
+        Base.openURL(url);
+      });
+    }
+    else {
+      // if not accessible IDE, keep link the same EXCEPT that now the link text is translated!
+      retString += " " + format("<a href=\"{0}\">{1}</a>", url, label);
```

Comment:
I'd change this into `retString += format("<a href=\"{0}\">{1}</a><br/>", url, label);` to target the other comment

Reply:
This is what I did in code change for other comment

Reply:
Not sure if the extra space at the beginning is still needed though... Better leave it there and eventually tackle it in a refactor to uniform the `setButtonOrLink` methods

Reply:
I don't believe that space is - or ever was - needed. That space is not use in the library manager version.

I've been looking for a utils class to move that method to, but I haven't found a home for it.

Do you want it changed? sounds like no.